src/components/UI/Overlay/OverlayContent/ConfirmationContext.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FunctionComponent } from "react";
+import { useOverlayContext } from "../../../../common/context/OverlayContext";
+import { Button } from "../../Button";
+import { IconSuccess } from "../../Icon";
+import warningIcon from "../../../../../src/static/images/Alert/warning.png";
+import { useLockBodyScroll } from "../../../../common/hooks/useLockBodyScroll";
+
+export const enum MESSAGE_TYPE {
+  SUCCESS = "success",
+  WARNING = "warning",
+  NONE = "none",
+}
+
+export interface ConfirmationContextProps {
+  messageType: MESSAGE_TYPE;
+  title?: string;
+  children?: React.ReactNode;
```

Comment:
is it intentional `title` will have a chance/need to pass in customised title? or could we assume it will be what `MESSAGE_TYPE is`? then we don't have to define this prop